events/iq_board.py

```diff
@@ -0,0 +1,94 @@
+import discord
+from discord.ext import commands
+import utils.logging as log
+import utils.colours as colour
+import utils.values as value
+import utils.database as sql
+from utils.helpers import *
+
+
+class IQBoard(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+        self.iq_emoji = "<:WeSmart:682352802202386472>"
+        self.iq_channel = -1  # TODO: Create channel and change ID
+        self.iq_requirement = 7
+
+    @commands.Cog.listener()
+    async def on_reaction_add(self, reaction, user):
+        if str(reaction) == self.iq_emoji:
+            if reaction.count >= self.iq_requirement:
+                channel = self.client.get_channel(self.iq_channel)
+                message = reaction.message
+                # Making sure people can't IQ Board IQ Boards
+                if message.channel == channel:
+                    return
+                embed = self.create_iq_embed(message, reaction)
+                # Check if embed already exists in #iq-board
+                iq_embed = await self.find_iq_embed(embed, channel)
+                if iq_embed is not None:
+                    await iq_embed.edit(embed=embed)
+                else:
+                    await channel.send(embed=embed)
+
+    @commands.Cog.listener()
+    async def on_reaction_remove(self, reaction, user):
+        if str(reaction) == self.iq_emoji:
+            channel = self.client.get_channel(self.iq_channel)
+            message = reaction.message
+            # Making sure people can't IQ Board IQ Boards
+            if message.channel == channel:
+                return
+            new_embed = self.create_iq_embed(message, reaction)
+            old_embed = await self.find_iq_embed(new_embed, channel)
+
+            # If it does not exist anymore we'll send it again
+            if old_embed is None and reaction.count > self.iq_requirement:
+                await channel.send(embed=new_embed)
+            elif reaction.count == 0:
+                await old_embed.delete()
+            else:
+                await old_embed.edit(embed=new_embed)
+
+    @commands.Cog.listener()
+    async def on_message_delete(self, message):
+        channel = self.client.get_channel(self.iq_channel)
```

Comment:
Would it be possible to have the iq_channel object stored somewhere permanently, instead of getting it each time with get_channel?

Reply:
Done, it's inside of `on_ready` because putting it in `__init__` returns None